tests/acceptance/dex/idtokenprovider.go

```diff
@@ -44,48 +48,59 @@ func (p *dexIdTokenProvider) implicitFlow() (map[string]string, error) {
 	if err != nil {
 		return nil, err
 	}
-	if authorizeResp.StatusCode < 300 || authorizeResp.StatusCode > 399 {
-		return nil, fmt.Errorf("Authorize - response error: '%s' - %s", authorizeResp.Status, readRespBody(authorizeResp))
+
+	switch authorizeResp.StatusCode {
+	case http.StatusFound:
+	case http.StatusOK:
+	default:
+		return nil, fmt.Errorf("got unexpected response on authorize: %d - %s", authorizeResp.StatusCode, readRespBody(authorizeResp))
 	}
 
 	// /auth/local?req=qruhpy2cqjvv4hcrbuu44mf4v
-	loginEndpoint := authorizeResp.Header.Get("location")
-	if strings.Contains(loginEndpoint, "#.*error") {
-		return nil, fmt.Errorf("Login - Redirected with error: '%s'", loginEndpoint)
+	var loginEndpoint string
+	if authorizeResp.StatusCode == 200 {
```

Comment:
you already check this status code few lines above, why duplicate yourself? if you need this double check, consider comparing with `http.StatusOK` like you do above in the switch

Reply:
Few lines above I just validate a response, if the status code is unexpected I exit with an error.
Here I will change 200 to http.StatusOK. Is it blocking this PR?

Reply:
ofc it's not ;-)